src/components/menu-bar/author-info.jsx

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import {FormattedMessage, injectIntl} from 'react-intl';
+import UserAvatar from './user-avatar.jsx';
+
+import styles from './author-info.css';
+
+const AuthorInfo = ({
+    className,
+    projectTitle,
+    userId,
+    username
+}) => (
+    <div
+        className={classNames(
+            className
+        )}
+    >
+        <div
+            className={classNames(
+                styles.authorInfo
+            )}
+        >
+            <UserAvatar
+                className={styles.avatar}
+                userId={userId}
```

Comment:
I think we should be passing a full url as `src` from the outside rather than constructing the url from a `userId` here. The details of the path, and even the host are outside the responsibility of GUI. It's possible eventually an environment (offline?) would pass in a dataURI, for instance.  And in the immediate, we would want to use a different thumbnail hosts in local development, on staging, and in production.

Reply:
Do we want this to link to the author's profile? @carljbowman?

If so, we should accomplish it with an `onAuthorClick` (or something) prop, since we'd want that to be handled differently in the desktop editor.

Reply:
Could also circle back to this.